exonum-java-binding/core/rust/src/utils/services.rs

```diff
@@ -53,45 +63,96 @@ mod tests {
     use tempfile::{Builder, TempPath};
 
     #[test]
-    fn no_config() {
-        let res = load_enabled_services("nonexistent");
-        assert!(res.is_err());
+    fn missed_system_services_section() {
```

Comment:
At the moment it is unclear from method names if they test `parse_services` or the whole load definition. Could the thing under test be clarified in each case?